pkg/tracing/jaeger/config_yaml.go

```diff
@@ -4,165 +4,167 @@
 package jaeger
 
 import (
-	"net"
+	"log"
+	"math"
 	"net/url"
 	"os"
 	"strconv"
 	"strings"
 	"time"
 
-	"github.com/opentracing/opentracing-go"
-	"github.com/pkg/errors"
-	"github.com/uber/jaeger-client-go"
-	"github.com/uber/jaeger-client-go/config"
-	"gopkg.in/yaml.v2"
+	"go.opentelemetry.io/contrib/samplers/jaegerremote"
+	"go.opentelemetry.io/otel/attribute"
+	otel_jaeger "go.opentelemetry.io/otel/exporters/jaeger"
+	tracesdk "go.opentelemetry.io/otel/sdk/trace"
 )
 
+type ParentBasedSamplerConfig struct {
+	LocalParentSampled  bool `yaml:"local_parent_sampled"`
+	RemoteParentSampled bool `yaml:"remote_parent_sampled"`
+}
+
 // Config - YAML configuration. For details see to https://github.com/jaegertracing/jaeger-client-go#environment-variables.
 type Config struct {
-	ServiceName            string        `yaml:"service_name"`
-	Disabled               bool          `yaml:"disabled"`
-	RPCMetrics             bool          `yaml:"rpc_metrics"`
-	Tags                   string        `yaml:"tags"`
-	SamplerType            string        `yaml:"sampler_type"`
-	SamplerParam           float64       `yaml:"sampler_param"`
-	SamplerManagerHostPort string        `yaml:"sampler_manager_host_port"`
-	SamplerMaxOperations   int           `yaml:"sampler_max_operations"`
-	SamplerRefreshInterval time.Duration `yaml:"sampler_refresh_interval"`
-	ReporterMaxQueueSize   int           `yaml:"reporter_max_queue_size"`
-	ReporterFlushInterval  time.Duration `yaml:"reporter_flush_interval"`
-	ReporterLogSpans       bool          `yaml:"reporter_log_spans"`
-	Endpoint               string        `yaml:"endpoint"`
-	User                   string        `yaml:"user"`
-	Password               string        `yaml:"password"`
-	AgentHost              string        `yaml:"agent_host"`
-	AgentPort              int           `yaml:"agent_port"`
-	Gen128Bit              bool          `yaml:"traceid_128bit"`
+	ServiceName                        string                   `yaml:"service_name"`
+	Disabled                           bool                     `yaml:"disabled"`
+	RPCMetrics                         bool                     `yaml:"rpc_metrics"`
```

Comment:
As we spoke, do you think it would make sense to print a deprecation warning for options that are not applicable to new tracer?